doc/administrator/administrate.rst.mako

```diff
@@ -11,13 +11,13 @@ required.
 Server OGC
 ----------
 
-This is the description of an OGC server (WMS/WFS).
+This is the description of an server OGC (WMS/WFS).
 For one server we try to create only one request when it's possible.
 
 If you want to query the same server to get PNG and JPEG images,
-you should defined two OGC server. Attributes:
+you should defined two server OGC. Attributes:
```

Comment:
"... you should define two OGC servers." (I also agree with Fred that it should be "OGC server" instead of "server OGC")